msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -362,6 +363,7 @@ Icons used within the Teams app. The icon files must be included as part of the
 |---|---|---|---|
 |`outline`|2048 characters|✔️|A relative file path to a transparent 32x32 PNG outline icon. The border color must be white.|
 |`color`|2048 characters|✔️|A relative file path to a full color 192x192 PNG icon.|
+|`color32x32`|2048 character| | A relative file path to a full color PNG icon with transparent background. Size 32x32.|
```

Comment:
For consistency with previous entries in this doc (and to explain usage), how about:

> A relative file path to a full color 32x32 PNG icon with transparent background. Used when the app is pinned in Outlook and Microsoft 365 app.

(Looks like there's already precedent in manifest reference docs for modifying the description from schema for easier readability: the description for `color` here ("A relative file path to a full color 192x192 PNG icon") has been modified from the original [text in the schema description](https://github.com/microsoft/json-schemas/commit/4d1a21b02160b5890a3fdd85c0eb90ab2de2158d) ("A relative file path to a full color PNG icon. Size 192x192.")